src/components/textarea/TextArea.tsx

```diff
@@ -24,6 +24,8 @@ export interface ITextAreaOwnProps {
     disabledOnMount?: boolean;
 
     isAutosize?: boolean;
+
+    onChangeCallBack?: (event: React.KeyboardEvent<HTMLTextAreaElement>) => void;

```

Comment:
Why not just use `onChange` if you aren't using Redux?

Reply:
I added this function for the redux use case ( if you want to do something onChange without removing the redux behaviour.